plugins/session_notifier.rb

```diff
@@ -237,6 +270,32 @@ def send_text_to_dingtalk(session)
         print_status((body['errcode'] == 0) ? 'Session notified to DingTalk.' : 'Failed to send notification.')
       end
 
+      def send_text_to_gotify(session)
+        # https://gotify.net/docs/more-pushmsg
+        uri_parser = URI.parse(gotify_address)
+        message_text =
+        "Platform : #{session.platform}\n" \
+        "Tunnel : #{session.tunnel_to_s}\n" \
+        "Arch : #{session.arch}\n" \
+        "Info : > #{session.info ? session.info.to_s : nil}"
+        json_post_data = JSON.pretty_generate({
+          title: "#{session.platform}主机#{session.type}会话上线!",
+          message: message_text,
+          priority: 10
+        })
+        http = Net::HTTP.new(uri_parser.host, uri_parser.port)
+        if gotify_ssl == 1
+          http.use_ssl = true
+          http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
Session information should be treated as confidential. I don't think we want to be disabling SSL by default. This could be it's own option though.

Reply:
Ok，I will try to use cert to verify connections